crafting_combinator/script/rc.lua

```diff
@@ -140,13 +155,37 @@ function _M:find_ingredients_and_products(forced)
 end
 
 
+function _M:find_machines(forced)
+	local recipe, input_count = recipe_selector.get_recipe(self.entity, nil, defines.circuit_connector_id.combinator_input)
+
+	if self.recipe ~= recipe or forced then
```

Comment:
This should probably use the same condition as [here](https://github.com/theRustyKnife/CraftingCombinator/blob/10d307dcb87577feaedc96db3e7a10289f6da6ec/crafting_combinator/script/rc.lua#L107) to update when the input count changes.

Reply:
I had that first, but I was messing around with it when i tried to update it after a new machine was researched.